express-get-json/index.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const app = express();
+
+const grades = {
+  12: {
+    id: 12,
+    name: 'Tim Berners-Lee',
+    course: 'HTML',
+    score: 95
+  },
+  47: {
+    id: 47,
+    name: 'Brendan Eich',
+    course: 'JavaScript',
+    score: 100
+  },
+  273: {
+    id: 273,
+    name: 'Forbes Lindsay',
+    course: 'JavaScript',
+    score: 92
+  }
+};
+
+app.get('/api/grades', (req, res) => {
+  res.json(() => {
+    const array = [];
+    for (const key in grades) {
+      array.push(grades[key]);
+    }
+  });
```

Comment:
Please update this when you have time.

Reply:
i still dont get how i produced that screenshot